src/app/(sidebar)/transaction/submit/components/ErrorResponse.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  AccountRequiresMemoError,
+  BadResponseError,
+  NetworkError,
+} from "@stellar/stellar-sdk";
+
+import { Box } from "@/components/layout/Box";
+import { ValidationResponseCard } from "@/components/ValidationResponseCard";
+
+interface ErrorProps {
+  error: NetworkError & {
+    response: {
+      data: {
+        extras: {
+          result_codes: string;
+          result_xdr: string;
+        };
+      };
+    };
+  };
+}
+
+export const ErrorResponse = ({ error }: ErrorProps) => {
+  let message = "",
+    extras = null;
+  if (error instanceof AccountRequiresMemoError) {
+    message = "This destination requires a memo.";
+    extras = (
+      <Box gap="xs">
+        <div>
+          <div>Destination account:</div>
+          <div>{error.accountId}</div>
+        </div>
+        <div>
+          <div>Operation index:</div>
+          {error.operationIndex}
+        </div>
```

Comment:
<img width="694" alt="Screenshot 2024-06-14 at 5 01 54 PM" src="https://github.com/stellar/laboratory/assets/3912060/1eec19ef-9431-48ba-ae8e-3b9b4054d175">
I made it work with `<Box/>`, but I had to add an extra empty `div` which makes me think i should just create a css style.

Reply:
Yeah, I think we should have a custom component for this because we have a similar layout in other places. We can still use the `Box` as a container; the label element can be a simple `div`, but the value should have a class to set the padding/margin from the left. :+1: